### PR TITLE
added css to admin panel and created add new project button and site

### PR DESCRIPTION
**Pull request by lucyholubova**

### CODE REVIEW

File: portfolio-edit/Panel.php [hunk 1]
```diff
@@ -10,16 +10,25 @@
...
     <meta charset="UTF-8">
+<link rel="stylesheet" href="panel.css">
 <body>
 
 <h1> Admin page </h1>
+
+<body>
+<?php
+echo '
+<div class="addNewProject"*> <a href="panel.html"><input type="submit" value="add new project"></a></div>';
```
Comment (olyolyoly) [inline portfolio-edit/Panel.php]: what is *?

**Review by olyolyoly — CHANGES_REQUESTED** [review]

File: portfolio-edit/Panel.php (hunk 1 above)
Comment (lucyholubova) [reply portfolio-edit/Panel.php]: fixed that

**Review by olyolyoly — COMMENTED** [review]
looks good :)

File: portfolio-edit/Panel.php [hunk 2]
```diff
@@ -10,67 +10,43 @@
...
     <meta charset="UTF-8">
+<link rel="stylesheet" href="panel.css">
 <body>
 
 <h1> Admin page </h1>
+
+<body>
+<?php
+echo '
+<div class="addNewProject"> <a href="panel.html"> add new project</a></div>';
```
Comment (mporam) [inline portfolio-edit/Panel.php]: why is this echo'd? you can just remove the php here and have normal html

File: portfolio-edit/Panel.php [hunk 3]
```diff
@@ -10,67 +10,43 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+<link rel="stylesheet" href="panel.css">
 <body>
 
 <h1> Admin page </h1>
+
+<body>
```
Comment (mporam) [inline portfolio-edit/Panel.php:16]: dont need a body tag here. not allowed 2 body tags

File: portfolio-edit/add-new-project.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
```
Comment (mporam) [inline portfolio-edit/add-new-project.php]: please remove unnecessary comments

File: portfolio-edit/add-new-project.php [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 2019-03-19
+ * Time: 11:26
+ */
+
+
+
+//var_dump($_POST);
```
Comment (mporam) [inline portfolio-edit/add-new-project.php]: please remove

File: portfolio-edit/add-new-project.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=Project_2', 'root', '');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+$sql = "INSERT INTO `portfolio` (`name`, `url`, `comment`) VALUES (:projectName, :projectUrl, :projectComment);";
+
+$query = $db->prepare($sql);
+$query->execute([
+    ':projectName' => $_POST['pname'],
```
Comment (mporam) [inline portfolio-edit/add-new-project.php]: what if pname, url or comment are empty? no check if they have any data in them

File: portfolio-edit/panel.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
```
Comment (mporam) [inline portfolio-edit/panel.html:3]: no body tag?

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio-edit/Panel.php (hunk 3 above)
Comment (lucyholubova) [reply portfolio-edit/Panel.php:16]: done

File: portfolio-edit/add-new-project.php (hunk 4 above)
Comment (lucyholubova) [reply portfolio-edit/add-new-project.php]: done

File: portfolio-edit/panel.html (hunk 7 above)
Comment (lucyholubova) [reply portfolio-edit/panel.html:3]: sorry

File: portfolio-edit/add-new-project.php (hunk 5 above)
Comment (lucyholubova) [reply portfolio-edit/add-new-project.php]: done

File: portfolio-edit/add-new-project.php (hunk 6 above)
Comment (lucyholubova) [reply portfolio-edit/add-new-project.php]: ok done

File: portfolio-edit/Panel.php (hunk 3 above)
Comment (lucyholubova) [reply portfolio-edit/Panel.php:16]: cool

File: portfolio-edit/add-new-project.php (hunk 6 above)
Comment (lucyholubova) [reply portfolio-edit/add-new-project.php]: ok

File: portfolio-edit/Panel.php (hunk 2 above)
Comment (lucyholubova) [reply portfolio-edit/Panel.php]: ok

**Review by mporam — APPROVED** [review]

**Pull request merged**